### ISSUE TITLE
Error: transaction expired

### ISSUE THREAD

**Issue by wd021** [iron-fish/ironfish#1372]
### What happened?

Seeing this in a few of my nodes

```
Peer zmpG/3C (azpwnz) sent orphan 00000...2ba72 (1874), syncing orphan chain.
Added block seq: 1660, hash: 00000...6fe4d, txs: 6, progress: 90.02%, time: 351.9ms
Requesting 20 blocks starting at 00000...2909a (1661) from XNDGkIR
Peer u/dy6Mh (roxa) sent orphan 00000...d0ce9 (1874), syncing orphan chain.
/root/ironfish/ironfish/build/src/blockchain/blockchain.js:605
                    throw new Error(verification.reason);
                          ^

Error: Transaction expired
    at /root/ironfish/ironfish/build/src/blockchain/blockchain.js:605:27
    at async LevelupDatabase.withTransaction (/root/ironfish/ironfish/build/src/storage/database/database.js:39:28)
    at async Blockchain.newBlock (/root/ironfish/ironfish/build/src/blockchain/blockchain.js:555:16)
    at async MiningManager.createNewBlockTemplate (/root/ironfish/ironfish/build/src/mining/manager.js:93:26)
    at async streamNewBlockTemplate (/root/ironfish/ironfish/build/src/rpc/routes/mining/blockTemplateStream.js:84:33)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
root@ip-172-31-44-60:~/ironfish/ironfish-cli# yarn start start
```

### Version
0.1.32

**Comment by dguenther** [comment]
I believe that was fixed in 0.1.33, can you try upgrading?

**Comment by wd021** [comment]
yup, haven't seen this since. upgraded a while ago.
